src/test/ui/consts/const-mut-refs/const_mut_address_of.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0658]: references in constants may only refer to immutable values
```

Comment:
Now is a good time to change this error message. I think we can call out `&mut` explicitly right? `&Cell<i32>` is a different structured error?

We need to explains the

Reply:
what about "`&mut` can only be used in `const fn`s"?

Reply:
Sounds good.